account_bank_statement_import/__openerp__.py

```diff
@@ -1,18 +1,16 @@
 # -*- encoding: utf-8 -*-
-# noqa: This is a backport from Odoo. OCA has no control over style here.
-# flake8: noqa
 {
     'name': 'Account Bank Statement Import',
-    'category' : 'Accounting & Finance',
+    'category': 'Accounting & Finance',
     'version': '1.0',
     'author': 'OpenERP SA',
     'depends': ['account'],
     'demo': [],
-    'description' : """Generic Wizard to Import Bank Statements.
-    
+    'description': """Generic Wizard to Import Bank Statements.
```

Comment:
As we are going to diverge from master, please put this on a README.rst with the proper OCA format.